Components/Navigation.css

```diff
@@ -0,0 +1,44 @@
+.header {
+    display: flex;
+    align-items: center;
+    flex-wrap: wrap;
+    padding: 0 8px;
+    background: var(--danger-message-color);
+  }
+  
+  .logo {
+    font-size: 20px;
+    font-weight: bold;
+  }
+  
+  .header__nav {
+    margin-left: auto;
+    display: flex;
+    align-items: center;
+    flex-wrap: wrap;
+  }
+  
+  .header__nav li {
+    display: inline-block;
+  }
+  .header__nav a {
+    padding: 8px;
+  }
+  
+  .user {
+    display: flex;
+    align-items: center;
+  }
+  .user .img-navigation {
+    width: 40px;
```

Comment:
- you can use here dynamic units like rem/em instead of px, by the way, it is up to you how you add responsiveness to your website (suggestion)
- Rest is good and clean
- Code formatting is nice ✅